### PR TITLE
Init AgFoodPlatform Mgmt SDK

### PR DESCRIPTION
**Pull request by priyankajayaswal1**
# Contributing to the Azure SDK

Creating Init SDK for .NET for public preview requirement for RP AgFoodPlatform

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/agrifood/ci.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ trigger:
   paths:
     include:
     - sdk/agrifood/
+    - sdk/resourcemanager/
```
Comment (ArthurMa1978) [inline sdk/agrifood/ci.yml:11]: please dont create a new ci.mgmt.yml file, refer this https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/advisor/ci.mgmt.yml

File: sdk/agrifood/Azure.ResourceManager.AgFoodPlatform/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("AgFoodPlatform")]
```
Comment (ArthurMa1978) [inline sdk/agrifood/Azure.ResourceManager.AgFoodPlatform/src/Properties/AssemblyInfo.cs:6]: Microsoft.AgFoodPlatform

File: sdk/agrifood/Azure.ResourceManager.AgFoodPlatform/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Azure AgFoodPlatform Management client library for .NET
```
Comment (ArthurMa1978) [inline sdk/agrifood/Azure.ResourceManager.AgFoodPlatform/README.md:1]: AgFood Platform

File: sdk/agrifood/Azure.ResourceManager.AgFoodPlatform/src/autorest.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: AgFoodPlatform
+namespace: Azure.ResourceManager.AgFoodPlatform
+require: https://github.com/Azure/azure-rest-api-specs/tree/e31e3938529269e0e6a81f60b2fdc6d2aec5b9df/specification/agrifood/resource-manager/readme.md
+tag: package-preview-2021-09
```
[the remark this answers is not in the document]
Comment (priyankajayaswal1) [reply sdk/agrifood/Azure.ResourceManager.AgFoodPlatform/src/autorest.md:12]: by this month end.

File: sdk/agrifood/Azure.ResourceManager.AgFoodPlatform/src/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (priyankajayaswal1) [reply sdk/agrifood/Azure.ResourceManager.AgFoodPlatform/src/Properties/AssemblyInfo.cs]: updated

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**